CardsCashCasino/Manager/TexasHoldEmManager.cs

```diff
@@ -470,6 +475,50 @@ public void Update()
 
             int totalPotValue = _potManager.Pots.Sum(pot => pot.Total); // Calculate total pot value
             _potUI.UpdatePot(totalPotValue);
+            
+            // Update the position of the turn indicator depending on who's turn it is
+            UpdateTurnIndicatorPosition();
+        }
+
+        /// <summary>
+        /// Called in the update loop. Updates the position of the turn indicator depending on who's turn it is.
+        /// </summary>
+        private void UpdateTurnIndicatorPosition()
+        {
+            int xPos = 0, yPos = 0;
+            
+            // if it's the user's turn, the indicator should appear next to the user's stack amount
+            switch (playerIndex)
+            {
+             case 0:
+                xPos = (Constants.WINDOW_WIDTH / 2) - 90; // This is shifted 15 pixels from the user's stack amount display
```

Comment:
We use the window size for the first case but not any of the others. These should be calculated given the window size constants, not just arbitrarily set. This causes scaling issues, which I was having with my macbook.

Reply:
The pixels are hardcoded because they come from the values that are used explicitly for all of the indicators in the game. Let's handle this at the back end of the sprint if there's time. I'm triaging the project and there's a lot to do with Five Card Draw.